src/args/lint-clojure/builder.sh

```diff
@@ -0,0 +1,14 @@
+# shellcheck shell=bash
+
+function main {
+  local args=(
+    --lint
+  )
+
+  info Compiling Clojure solution \
+    && clj-kondo "${args[@]}" "${envTarget}" \
+    || critical Unsuccessful compilation \
```

Comment:
```suggestion
```

we can add these messages at the CLI level, let's avoid adding them here

Reply:
you mean add this to shell-commands, right?

Reply:
no, to src/cli/main/__main__.py

in other words just remove the line, I'll add the 'success/fail' message later